### DISCUSSION TITLE
Client secrets are inaccessible for users who signed up with Apple (and probably all third-party login)

### DISCUSSION THREAD

**Discussion by tony-pizza** [tidal-music/discussions#44] [Report bug]
I signed up for Tidal using Apple login and I created a developer application in my Tidal account. However, when I try to view/copy the client secret for my application Tidal requires that I enter my password. Because I signed up with Apple, I don't have a password, and so I cannot access my client secret. I haven't checked, but I assume this behavior would impact other third-party logins like Google and Facebook.

**Comment by tony-pizza** [comment]
Ah, I'm noticing there's a moment when I can view/copy the secret before it becomes hidden. I will just create another application and grab the secret before it disappears. Leaving this discussion open as the password protection still effectively leads to broken UX for third-party logins.

**Reply by stefanmoro** [reply]
Thanks @tony-pizza,

Let us look into it, and we'll update here.

**Reply by aliaksei-taliuk** [reply]
@tony-pizza As a temporary workaround you can try to go to http://login.tidal.com/resetpass and set a password for your TIDAL account and use it for addressing this problem... Meantime we'll try to think on some better long term solution...
